Kernel/Sections.h

```diff
@@ -13,11 +13,15 @@
 
 #define KERNEL_BASE 0xC0000000
 #define KERNEL_PD_OFFSET 0x3000000
-#define KERNEL_PD_END 0xF1000000
-#define KERNEL_PT1024_BASE 0xFFE00000
+#define KERNEL_PD_END (KERNEL_BASE + 0x31000000)
+#define KERNEL_PT1024_BASE (KERNEL_BASE + 0x3FE00000)
 #define KERNEL_QUICKMAP_PT (KERNEL_PT1024_BASE + 0x6000)
 #define KERNEL_QUICKMAP_PD (KERNEL_PT1024_BASE + 0x7000)
 #define KERNEL_QUICKMAP_PER_CPU_BASE (KERNEL_PT1024_BASE + 0x8000)
 #define KERNEL_PHYSICAL_PAGES_BASE (KERNEL_BASE + KERNEL_PD_OFFSET)
 
+#ifdef __cplusplus
+static_assert(KERNEL_BASE % 0x1000000 == 0);
```

Comment:
```suggestion
static_assert(KERNEL_BASE % 0x40000000 == 0);
```

The kernel base address should be aligned to 1GB - at least for now. My bad. :)

Reply:
Nevermind, the next series of patches removes this again, so it doesn't matter.